generators/generator-base.js

```diff
@@ -808,7 +808,28 @@ module.exports = class extends PrivateBase {
      * Generate a date to be used by Liquibase changelogs.
      */
     dateFormatForLiquibase() {
-        const now = new Date();
+        let now = new Date();
+        // Run reproducible timestamp when regenerating the project with with-entities option.
+        if (this.options.withEntities) {
```

Comment:
this is so complex that I think this deserves to be in its own function, your opinion?
It could be called getValueLastLiquibaseTimestamp

Reply:
I think it became simple enough now.